data.json

```diff
@@ -98,6 +98,15 @@
     "dateAdded": "2018-10-12T13:25:10.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers"]
   },
+  {
+    "name": "JS Bin",
+    "difficulty": "easy",
+    "description": "Fill out a Google form with your name and address and get a sticker. There’s no cost to you, but they will be restricting the send outs to bi-monthly or quaterly depending on demand.",
+    "reference": "https://jsbin.com/help/stickers/",
+    "image": "https://help.jsbin.com/images/stickers.jpg",
```

Comment:
Let's change this image for:


![2019-01-30-150059_349x383_scrot](https://user-images.githubusercontent.com/8191198/51986158-eb719400-249f-11e9-9141-f8eb3aec2bd8.png)


```suggestion
    "image": "https://pbs.twimg.com/media/CccQEDjUUAE9wRO.jpg",
```